README.md

```diff
@@ -820,6 +825,9 @@ Robotics: Science and Systems XIV (RSS), 2019. [PDF](https://www.research-collec
 - <a name="Chen19fnins"></a>Chen, G., Chen, J., Lienen, M., Conradt, J., Roehrbein, F., Knoll, A.C.,  
 *[FLGR: Fixed Length Gists Representation Learning for RNN-HMM Hybrid-Based Neuromorphic Continuous Gesture Recognition](https://dx.doi.org/10.3389%2Ffnins.2019.00073)*,  
 Front. Neurosci., 2019, 13:73.
+- <a name="Tapiador18tbiocas"></a>Tapiador-Morales, R., Linares-Barranco, A., Jimenez-Fernandez, A., Jimenez-Moreno, G. 
+*[Neuromorphic LIF Row-by-Row Multiconvolution Processor for FPGA](https://doi.org/10.1109/TBCAS.2018.2880012)*,  
+IEEE Transactions on Biomedical Circuits and Systems, 2019, vol. 13, issue 1.
```

Comment:
Can you abbreviate the name of the journal as in IEEE convention?
http://technicalghostwriters.com/wp-content/uploads/2012/02/IEEE-Abbreviations-for-Transactions-Journals-Letters.pdf
thanks